includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -619,14 +619,19 @@ public function header_logo() {
 	}
 
 	public function get_settings_text( $settings_key, $default = false, $autop = true ) {
-		if ( !empty( $this->settings[$settings_key]['default'] ) ) {
+		if( array_key_exists( 'default', $this->settings[$settings_key] ) ) {
```

Comment:
this changes the current behavior of using the default when nothing is filled in. While that could be an improvement, it's a breaking change so I'd prefer to stick to the current method. You'll need both.

Reply:
you marked this as resolved but you didn't reintroduce the `!empty( $this->settings[$settings_key]['default'] )` condition.